Mesh_3/include/CGAL/IO/facets_in_complex_3_to_triangle_mesh.h

```diff
@@ -28,37 +28,33 @@
 #include <CGAL/array.h>
 #include <CGAL/boost/graph/Euler_operations.h>
 #include <CGAL/Hash_handles_with_or_without_timestamps.h>
+#include <CGAL/Polygon_mesh_processing/orientation.h>
```

Comment:
FYI, this will add a dependency to PMP-orientation. Not sure it was there before. The soup -> triangle mesh might be left to the user is we want to avoid that.

Reply:
No, this function must return a `triangle_mesh`, it is called `_to_triangle_mesh`.

Besides, `orientation.h` is not needed for this conversion, only `orient_polygon_soup.h` and `polygon_soup_to_polygon_mesh.h`. `orientation.h` was only added because I added
```
  if(CGAL::is_closed(graph) && !PMP::is_outward_oriented(graph))
    PMP::orient_to_bound_a_volume(graph);
```
which can certainly be removed (I just got a bit annoyed that it seemed to always pick the wrong orientation and didn't think about dependencies).

Reply:
Why is that an issue to depend on `PMP::orient_to_bound_a_volume`?